src/RoadRegistry.Legacy.Extract/Readers/ImportedMunicipalitiesReader.cs

```diff
@@ -0,0 +1,92 @@
+namespace RoadRegistry.Legacy.Extract.Readers
+{
+    using System;
+    using System.Collections.Generic;
+    using BackOffice.Framework;
+    using BackOffice.Messages;
+    using Be.Vlaanderen.Basisregisters.Shaperon;
+    using Be.Vlaanderen.Basisregisters.Shaperon.Geometries;
+    using Microsoft.Data.SqlClient;
+    using Microsoft.Extensions.Logging;
+    using NodaTime;
+    using NodaTime.Text;
+    using Point = BackOffice.Messages.Point;
```

Comment:
Please remove this and favour full qualification - it makes it more readable and explicit at the location the type is used.